homeassistant/components/plugwise/sensor.py

```diff
@@ -31,27 +32,49 @@
         native_unit_of_measurement=TEMP_CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
```

Comment:
Why is this one disabled?

Reply:
The Setpoint value is present in the climate card so the sensor is double.

Reply:
I prefer to remove the entity-category.

Reply:
Same as above, this change is incorrect and should be reverted.

Reply:
Reverted.